otcextensions/tests/functional/sdk/dms/v1/test_message.py

```diff
@@ -26,78 +26,76 @@ class TestMessage(base.BaseFunctionalTest):
     messages = []
     received_messages = []
 
-    @classmethod
-    def setUpClass(cls):
-        super(TestMessage, cls).setUpClass()
+    def setUp(self):
+        super(TestMessage, self).setUp()
         openstack.enable_logging(debug=True, http_debug=True)
         try:
-            cls.queue = cls.conn.dms.create_queue(
+            self.queue = self.conn.dms.create_queue(
                 name=TestMessage.QUEUE_ALIAS
             )
 
         except openstack.exceptions.BadRequestException:
-            cls.queue = cls.conn.dms.get_queue(TestMessage.QUEUE_ALIAS)
+            self.queue = self.conn.dms.get_queue(TestMessage.QUEUE_ALIAS)
 
-        cls.queues.append(cls.queue)
+        self.queues.append(self.queue)
```

Comment:
shouldn't this be TestMessage.queues.append ?

Reply:
Nope, we want to add to object queue, and not class queue

Reply:
Ignore the comment :D

Reply:
> 
> 
> Nope, we want to add to object queue, and not class queue

Sorry i didnt see your feedback on my comment and then wrote another comment, but thank you for clearing 👍